full-v1.yml

```diff
@@ -1383,7 +1383,7 @@ definitions:
         x-validation: true
       dob:
         type: string
-        format: datetime
+        pattern: (?:[0-9]{1,2})/([0-9]{1,2})/([0-9]{4})
```

Comment:
This is mm/dd/yyyy which is different from your PR description. Just double checking this was intended.

Reply:
Nvm I read the PR description backwards!

Reply:
Oh good point - let me confirm

Reply:
My PR description is wrong, we use 4 digits for the date